app/Providers/Macros/Response/ToProjectMobileLogoS3Macro.php

```
@@ -0,0 +1,79 @@
+                } catch (FileNotFoundException $e) {
+                    Log::error('Cannot find S3 project mobile logo', ['exception' => $e]);
+
+                } catch (Throwable $e) {
+                    Log::error('Cannot generate S3 project mobile logo', ['exception' => $e]);
+                }
+            }
+
+            // Default placeholder
+            $file = Storage::disk('public')->get($photoPlaceholderFilename);
+            // Read bytes and create project mobile logo
+            $image = Image::read($file);
```

PHPMD (2.15.0)

Avoid using static access to class '\Intervention\Image\Laravel\Facades\Image' in method 'boot'. (Clean Code Rules)

(StaticAccess)